admin-frontend/open_admin_app/lib/widgets/user/list/list_users_bloc.dart

```diff
@@ -17,17 +17,26 @@ class ListUsersBloc implements Bloc {
   final ManagementRepositoryClientBloc mrClient;
   final PersonServiceApi _personServiceApi;
 
+  bool isPerson;
+
+  final _adminApiKeysSearchResultSource =
+      BehaviorSubject<List<Person>>.seeded([]);
+
+  Stream<List<Person>> get adminAPIKeySearch =>
+      _adminApiKeysSearchResultSource.stream;
+
   Stream<List<SearchPersonEntry>> get personSearch =>
       _personSearchResultSource.stream;
   final _personSearchResultSource =
       BehaviorSubject<List<SearchPersonEntry>>.seeded([]);
 
-  ListUsersBloc(this.search, this.mrClient)
+  ListUsersBloc(this.search, this.mrClient, this.isPerson)
       : _personServiceApi = PersonServiceApi(mrClient.apiClient) {
-    triggerSearch(search);
+    triggerSearch(search, true);
+    triggerSearch(search, false);
```

Comment:
this is a very heavy load on the server, i'll take a note and fix this because we can't have this